Packs/SkyhighSecurity/Integrations/SkyhighSecurity/SkyhighSecurity.py

```diff
@@ -28,6 +29,18 @@
 ''' CLIENT CLASS '''
 
 
+def csv2json(csv_data: str):
+    """ Converts data from csv to json
+    Args:
+        csv_data: data in csv format
+    Returns:
+        the same data in json formal
+    """
+    reader = csv.DictReader(io.StringIO(csv_data))
+    json_data = list(reader)
+    return json_data
+
+
```

Comment:
It looks like instead of a normal JSON parser we had until now, we are changing to using csv.DictReader, this means the command never worked?

Reply:
Yes, never worked before.
The response headers are in text/csv.
Not sure how it wasn't caught before the first merge

Reply:
@MosheEichler FYI

Reply:
As @Shellyber says, the response appears to be in CSV format. However, the function may work when there are responses, but in this case, it appears that there are none (content length is 0).